sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner_test.py

```diff
@@ -2110,6 +2111,30 @@ def process(
         if expected_groups:
           assert_that(grouped, equal_to(expected_groups), label='CheckGrouped')
 
+  def test_time_based_split_manager(self):
+
+    elements = list(range(100))
+
+    class BundleCountingDoFn(beam.DoFn):
+      def process(self, element) -> Iterator[object]:
+        time.sleep(0.001)
+        yield element
+
+      def finish_bundle(self):
+        yield window.GlobalWindows.windowed_value('endOfBundle')
+
+    with self.create_pipeline() as p:
+      p._options.view_as(DirectOptions).test_splits = {
+          'SplitMarker': {
+              'timings': [0, .05], 'fractions': [0.5, 0.5]
+          }
+      }
+      assert_that(
+          p
+          | beam.Create(elements)
+          | 'SplitMarker' >> beam.ParDo(BundleCountingDoFn()),
+          equal_to(elements + ['endOfBundle'] * 2))
```

Comment:
I think you answer this below:
```
            // More than one "endOfBundle" output because we split.
            // (We in fact should have split multiple times, but all the
            // remainders get concatenated into the second bundle.
            // Importantly, however, we verify that regardless of where the
            // splits occurred (if you print them, they're in an interesting
            // order), all elements are accounted for.)
```

Though I'm not sure why they get concatenated. Regardless, dropping a similar comment here would be helpful

Reply:
Yeah. Done.